java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/injection/RootModule.java

```diff
@@ -33,10 +33,15 @@
 
   private final String jarRuntimeImage;
   private final String serverRuntimeImage;
+  private final String tomcatRuntimeImage;
 
-  public RootModule(String jarRuntimeImage, String serverRuntimeImage) {
+  /**
+   * Constructs a new {@link RootModule} for Guice.
```

Comment:
Since you're already in the neighborhood, mind adding the `@param` information to the javadoc here? I think it would be great to have a brief description of each of these parameters.